packages/cozy-client/src/models/document/emojiCountry.js

```diff
@@ -1,14 +1,26 @@
-const emojiCountry = {
-  fr: '🇫🇷'
-}
+import logger from '../../logger'
+
+// The offset between uppercase ASCII and regional indicator symbols
+const offset = 127397
```

Comment:
comme c'est une constante on pourrait l'écrire un `OFFSET` et peut-être trouver un nom moins générique 🤔

Reply:
Ok, let's go to `UNICODE_OFFSET`.